keepassxc-browser/background/browserAction.js

```diff
@@ -47,7 +47,7 @@ browserAction.showDefault = async function(tab) {
         tab = tabs[0];
     }
 
-    if (page.tabs[tab.id] && page.tabs[tab.id].loginList.length > 0) {
+    if (page.tabs[tab.id]?.loginList.length > 0) {
```

Comment:
I _think_ these should also have a `?` before `.length`, i.e.:

```javascript
    if (page.tabs[tab.id]?.loginList?.length > 0) {
```

Otherwise this will happen:
```
({})?.loginList.length > 0
Uncaught TypeError: (intermediate value).loginList is undefined
```

Reply:
Could you add this to #1983? Thanks.

Reply:
Sure 👍

Reply:
You know, I think I was wrong after all. Thanks to the [short circuiting feature](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#short-circuiting) in optional chaining, this is safe. My example was flawed, this is more accurate:

```javascript
({})[1234]?.loginList.length > 0
false
```